dj_crm/data/equalizer.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+  <record model="dj.equalizer" id="dj_equalizer_mail_alias">
+    <field name="model">mail.alias</field>
+    <field name="xmlid_fields">alias_model_id,alias_parent_model_id,alias_user_id,alias_name</field>
```

Comment:
alias_parent_model_id is needed? Isn't it unlikely that we  have the same name for an alias on a different model on the same user?